server/e2e_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/cashshuffle/cashshuffle/message"
 
 	"github.com/avast/retry-go"
+	log "github.com/sirupsen/logrus"
```

Comment:
NIT: be consistent with placement, maybe put named imports at the bottom?

Reply:
It is easier to read that way.

1da862a

Reply:
.......  450aa8b